rest/members.go

```diff
@@ -40,8 +40,9 @@ func (s *memberImpl) GetMember(guildID snowflake.ID, userID snowflake.ID, opts .
 	return
 }
 
-func (s *memberImpl) GetMembers(guildID snowflake.ID, opts ...RequestOpt) (members []discord.Member, err error) {
-	err = s.client.Do(GetMembers.Compile(nil, guildID), nil, &members, opts...)
+func (s *memberImpl) GetMembers(guildID snowflake.ID, limit int, opts ...RequestOpt) (members []discord.Member, err error) {
+	reqOptions := append(opts, WithQueryParam("limit+", limit))
```

Comment:
you can pass a map of query params directly to compile. and there is a `+` which isn't needed

Reply:
i updated this in the most recent commit